tests/benchmark.py

```diff
@@ -15,14 +15,19 @@
 else:
     name_attr = '__name__'
 
+if hexversion < 0x03030000:
+    clock = time.clock
+else:
+    clock = time.perf_counter
```

Comment:
```suggestion
clock = time.clock if hexversion < 0x03030000 else time.perf_counter
```

Reply:
I used the expanded form to match the code style above at line 12
Do you think both must be changed?

Reply:
I see. Never mind.